src/toastify.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
And, this is the file causing problems.
I believe there were some dependabot updates failing because of this. 

Thanks!